wb_vandalism/features/feature.py

```diff
@@ -26,3 +27,14 @@ def __init__(self, property):
 
     def _process(self, changed_claims):
         return self.property in [claims[0].id for claims in changed_claims]
+
+
+class has_in_comment(Modifier):
+    def __init__(self, check):
+        self.check = check
+        name = "has_in_comment({0})".format(repr(check))
+        super().__init__(name, self._process, returns=bool,
+                         depends_on=[comment])
+
+    def _process(self, comment):
+        return self.check in comment
```

Comment:
I thought it too. but then I got stuck into issue of performance vs. accuracy. It's almost impossible to be at the middle of an edit summary. And we can use startswith() instead.
If you think it doesn't really matter. I'd love to change it :)